js_modules/dagster-ui/packages/ui-core/src/assets/AssetTable.tsx

```diff
@@ -101,16 +96,9 @@ export const AssetTable = ({
               icon="search"
               title="No matching assets"
               description={
-                isFiltered ? (
```

Comment:
These two varying messages were nice, would it make sense to implement isFiltered based on whether or not there is an asset selection syntax string?

Reply:
it's always filtered in this branch. Theres a no assets message further up the tree